frontend/src/toolbar/button/ToolbarButton.scss

```diff
@@ -1,98 +1,156 @@
+// default to light theme
 #button-toolbar {
-    position: fixed;
-    left: 0;
-    top: 0;
-    z-index: 2147483020;
-    opacity: 1;
-    transition: opacity ease 0.5s;
+    --text-3000: var(--text-3000-light);
```

Comment:
Ideally we could re-use the `global.scss` variables to make sure there isn't a drift across versions but that's probably beyond the scope here 🤷

Reply:
Moved some things into variables so they could be shared here